pkg/util/metadata/metadata.go

```diff
@@ -101,7 +101,7 @@ func getConfigDrivePath(metadataVersion string) string {
 	return fmt.Sprintf(configDrivePathTemplate, metadataVersion)
 }
 
-func getMetadataFromConfigDrive(metadataVersion string) (*Metadata, error) {
+func GetMetadataFromConfigDrive(metadataVersion string) (*Metadata, error) {
```

Comment:
The method name is a little bit redundant, could we just change it to something like `getFromConfigDrive`?